src/frontend/js/components/SaleTunnelStepValidation/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { defineMessages, FormattedMessage, FormattedNumber } from 'react-intl';
+import { Priority } from 'types';
+import type * as Joanie from 'types/Joanie';
+import CourseRunsList from './CourseRunsList';
+
+const messages = defineMessages({
+  includingVAT: {
+    defaultMessage: 'including VAT',
+    description: 'Text displayed next to the price to mention this is the price including VAT.',
+    id: 'components.SaleTunnelStepValidation.includingVAT',
+  },
+  proceedToPayment: {
+    defaultMessage: 'Proceed to payment',
+    description: 'CTA to go to payment step',
+    id: 'components.SaleTunnelStepValidation.proceedToPayment',
+  },
+});
+
+interface SaleTunnelStepValidationProps {
+  next: () => void;
+  product: Joanie.Product;
+}
+
+export const SaleTunnelStepValidation = ({ product, next }: SaleTunnelStepValidationProps) => {
+  const isOpenedCourseRun = (courseRun: Joanie.CourseRun) =>
+    courseRun.state.priority <= Priority.FUTURE_NOT_YET_OPEN;
+
+  return (
+    <section className="SaleTunnelStepValidation">
+      <header className="SaleTunnelStepValidation__header">
+        <h3 className="SaleTunnelStepValidation__title">{product.title}</h3>
+        <h4 className="SaleTunnelStepValidation__price">
```

Comment:
After seeing it in the UI I'd insist we could do better here by replacing this h4 by another tag (whatever: div, p, strong..), as it's more of a visual title than a title sectioning content.

Reply:
I think we could improve headings a little further, but we can do this after merging I guess. It's an issue that the modal begins with `h6` with the steps, then bumps back to `h3`.

Since we are in a modal that has its own isolated context, the step title would be better as an `h1` (or `h2`, a using multiple `h1` is always a big debate, but in the end the important part is to have a coherent headings hierarchy), followed then by the `SaleTunnelStepValidation__title` as h2, etc.

Reply:
Right, I will replace this by a paragraph.